src/System.Linq.Parallel/src/System/Linq/Parallel/Channels/SynchronousChannel.cs

```diff
@@ -29,15 +29,7 @@ internal sealed class SynchronousChannel<T>
 #if DEBUG
         // In debug builds, we keep track of when the producer is done (for asserts).
         private bool _done;
-#endif
-
-        //-----------------------------------------------------------------------------------
-        // Instantiates a new queue.
-        //
-
-        internal SynchronousChannel()
```

Comment:
While this change looks like dead code it actually isn't, by removing this the compiler will actually provide a public default constructor for this type now. Arguably given this type is internal it doesn't really matter but I would leave the internal constructor.